simapp/simd/main.go

```diff
@@ -9,6 +12,7 @@ import (
 func main() {
 	rootCmd := cmd.NewRootCmd()
 	if err := svrcmd.Execute(rootCmd, "", simapp.DefaultNodeHome); err != nil {
-		panic(err)
+		log.NewLogger(rootCmd.OutOrStdout()).Error("failure when running app", "err", err)
```

Comment:
I still feel like we should display the error (which wasn't the case before the panic addition)

Reply:
Yes, this is correct. Most cobra apps will print the error to STDERR and then `os.Exit(1)`.